ruth/tools/simulator.py

```diff
@@ -42,9 +42,14 @@ class CommonArgs:
 @dataclass
 class AlternativesRatio:
     default: float
-    fastest_paths: float
-    shortest_paths: float
-    distributed: float
+    dijkstra_fastest: float
+    dijkstra_shortest: float
+    plateau_fastest: float
+
+    def __post_init__(self):
+        self.default = 1 - self.dijkstra_fastest - self.dijkstra_shortest - self.plateau_fastest
+        if self.default < 0:
```

Comment:
Probably this should check that the result is close to zero? Otherwise it could also be e.g. `0.5`.

Reply:
If default is 0.5, then 50 % of vehicles will not use alternatives. This is the intended behaviour, if that's what you meant.

Reply:
Ok, but then the error message shouldn't say `Sum of alternatives ratios must be equal to 1.`, that sounds like the sum has to really be `== 1`, not just `<= 1` :)